TAC/Meetings/2024/2024-02-08/CCC-TAC-Minutes-2024-02-08.md

```diff
@@ -0,0 +1,149 @@
+* Nate Waddington from CNCF introduced themselves.
+
+## Meeting Minute Approval & Old Business
+
+* DM reviewed the status of new project onboarding
+* DM discussed the status of the kernel working group and that this is ready to begin having scheduled meetings
+* Meeting minutes will be approved via Github.
+
+## New Business
+
+### CCC Glossary
+
+Sal Kimmich (SK) introduced the CSA glossary and that they were not able to easily pull definitions. SK stated that we can create a machine readable glossary. Alec Fernandez (AF) provided additional context. Mike Bursell (MB) voiced his support for a glossary page on the website. Discussion ensued.  Sal Kimmich (SK) discussed the work being done with LF research and that a visualization is part of that work, they went on to ask folks to submit their visualizaion ideas for consideration.
+ 
+DM asked about the status of the attestation Wikipeida page. MB stated that it has not been published and to be careful because Wikipedia pages are mutable. 
+
+Riaan Kleinhans shared the CNCF glossary. Nate Waddington (NW) stated that this site was community driven and that it was one of their highest trafficked websites.  Alec Fernandez (AF) asked how they manage controversial language. NW went over the CNCF process.
+
+MB stated that he thinks we need a terminology SIG and without this it will not move forward. DM proposed having the SIG work via a Github repository to avoid meetings and allow asynchronous work. AF suggested working to define a small number of terms to serve as an MVP. 
+
+MB volunteered to help define the first terms. AF stated that he would look through docs for terms but suggested the following to start with:
+
+TEE
+TCB (Trusted Computing Base)
+Attestation
+Root of trust
+Confidential Computing
+
+## TAC Priorities
+
+Yash Mankad (YM) stated that CCC does not have a formal mentorship program and thanked Riaan for helping him understand how CNCF manages their mentorship program. YM went on to discuss how CNCF handles matching mentees to mentorship opportunities. YM stated that their process is very standardized with specific terms throughout the year.
+
+Nate Waddington (NW) discussed the CNCF mentoring program and the CNCF Mentoring working group processes. Discussion ensued about the mentorship program and how it would be beneficial to the CCC.
+YM proposed that mentorship participation should be a requirement for Incubation projects to increase their contributor base.
+NW continued to discuss the Google Summer of Docs as a program to improve projects technical documentation, which is a Technical writer program and not mentorship.
+YM proposes ccc should target term 2 with a target of 3 or 4 projects to participate in the process in 2024.
+
+## 2024 TAC Objectives 
+DM explained the OKR process for 2024. 
+YM world leed community
+Catalin Sandu would lead on Ecosystem development
+Alex Fernandez would lead would work with the GRC and the TAC on defining definitions for industry
```

Comment:
Alec*
or AF ... as his name is mentioned previously on line 109

Reply:
Updated